lib/matplotlib/font_manager.py

```diff
@@ -1114,7 +1114,10 @@ def __init__(self, size=None, weight='normal'):
     def addfont(self, path):
         """
         Cache the properties of the font at *path* to make it available to the
-        `FontManager`.  The type of font is inferred from the path suffix.
+        `FontManager`. The type of font is inferred from the path suffix.
+
+        To cache globally and persistently, instead use
+        `matplotlib.font_manager.fontManager.addpath(path)`.
 
```

Comment:
Either use double backtick to create fixed-width text or remove the argument to get a link to the merhod. Putting a tilde first will abbreviate a the text to just the method.